### PR TITLE
Update scripts for Matlab R2022a and Vivado 2022.2

### PR DESCRIPTION
**Pull request by StancaPop**
Update the existing scripts to work with Matlab R2022a and Vivado 2022.2.

It has been successfully tested along with the Precision Toolbox.

### CODE REVIEW

File: build_bsp.sh [hunk 1]
```diff
@@ -72,75 +67,34 @@ do
...
+# Updates
+cp scripts_hdl/matlab_processors.tcl ../hdl/vendor/AnalogDevices/vivado/projects/scripts/matlab_processors.tcl
+cp scripts/system_project_rxtx.tcl ../hdl/vendor/AnalogDevices/vivado/projects/scripts/system_project_rxtx.tcl
+cp scripts/adi_build.tcl ../hdl/vendor/AnalogDevices/vivado/projects/scripts/adi_build.tcl
+
+# Copy fsbl files
+cp scripts/fsbl_build_zynq.tcl ../hdl/vendor/AnalogDevices/vivado/projects/scripts/fsbl_build_zynq.tcl
+cp scripts/fsbl_build_zynqmp.tcl ../hdl/vendor/AnalogDevices/vivado/projects/scripts/fsbl_build_zynqmp.tcl
+cp scripts/pmufw_zynqmp.tcl  ../hdl/vendor/AnalogDevices/vivado/projects/scripts/pmufw_zynqmp.tcl
+cp scripts/fixmake.sh  ../hdl/vendor/AnalogDevices/vivado/projects/scripts/fixmake.sh
```
Comment (tfcollins) [inline build_bsp.sh]: Not necessarily anymore if the adi_build.tcl script ignore them

File: fixmake.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#grep "CC_FLAGS :=" pmufw/Makefile | grep -e "-Os" || sed -i '/-mxl-soft-mul/ s/$/ -Os -flto -ffat-lto-objects/' pmufw/Makefile
+cd pmufw
+make
```
Comment (tfcollins) [inline fixmake.sh]: Can remove this file. Not necessary anymore

**Review by tfcollins — CHANGES_REQUESTED** [review]
Need some minor cleanup

File: build_bsp.sh (hunk 1 above)
Comment (StancaPop) [reply build_bsp.sh]: Deleted the unnecessary files.

File: fixmake.sh (hunk 2 above)
Comment (StancaPop) [reply fixmake.sh]: Removed.

**Comment by StancaPop** [comment]
Made the required cleanup. Fixed the BOOT.BIN generation and added the necessary boot files. Tested along with Precision Toolbox within the add_cn0585_fmcz branch.

**Pull request merged**